client/src/components/profile/components/TimeLine.js

```diff
@@ -1,13 +1,18 @@
-import React, { Component } from 'react';
+import React, { Component, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import format from 'date-fns/format';
-import { find, reverse, sortBy } from 'lodash';
+import find from 'lodash/find';
+import reverse from 'lodash/reverse';
+import sortBy from 'lodash/sortBy';
```

Comment:
Wondering if there's a reason that there were separated? Looked cleaner how it was to me.

Reply:
@moT01 I saw in some components that only needed functions were being imported from lodash and though the change would be brought to all components. Thats why i did like this.

Reply:
I thought both versions were doing the same thing.